op.c

```diff
@@ -10243,7 +10243,9 @@ Perl_newMYSUB(pTHX_ I32 floor, OP *o, OP *proto, OP *attrs, OP *block)
            my sub foo; sub { sub foo { } }
      */
   redo:
+    assert(pax);
```

Comment:
This is a (signed) integer array index, so I think `assert(pax > 0);` would make more sense.

Reply:
Good call. Now added.